gramps/gen/filters/test/filter_optimizer_test.py

```diff
@@ -207,6 +207,18 @@
         <arg value="Person I1041"/>
       </rule>
     </filter>
+    <filter name="Home Person" function="and">
+      <rule class="IsDefaultPerson" use_regex="False" use_case="False">
+      </rule>
+    </filter>
```

Comment:
```suggestion
    </filter>
    <filter name="People of Events of Home Person" function="and">
      <rule class="MatchesEventFilter" use_regex="False" use_case="False">
        <arg value="Events of Home Person"/>
      </rule>
    </filter>    
```

Add an additional filter where we go from Person -> Event -> Person